tools/boxr_config.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Should this be here? It seems potenitally private...

Reply:
It is "mildly" private in the sense somebody could use it to modify that accounts Box workspace. But that account is for testing purposes so it doesn't have anything valuable in it and I can always revoke tokens if one gets compromised.

I'm sure there is a better place to store sensitive things like this, but I wanted to share bc I was thinking this would be part of the Travis build/testing. I have limited experience with Travis build args/configuration but that was my next step after JWT.

Reply:
Glad to hear - on my list of things to do is to figure out a simpler way to encrypt tokens for testing and to remain CRAN-friendly. The current reference is this: <https://cran.r-project.org/web/packages/googlesheets/vignettes/managing-auth-tokens.html>, but there *has* to be a simpler way. 

Maybe along the lines being used for deploying pkgdown? <https://pkgdown.r-lib.org/reference/deploy_site_github.html>

Reply:
Just to note, the tidyverse team have come up with a cleaner way to manage credentials for testing: https://gargle.r-lib.org/articles/articles/managing-tokens-securely.html

I think that implementing this will be a separate issue, but just to note it here.